src/app/complex/complex-results/complex-results.component.css

```diff
@@ -36,12 +36,12 @@
 }
 
 .filterColumn {
-  width: calc(100rem * 0.22);
+  width: 15vw;
 }
 
 .leftAligned {
   --global-margin: calc((100vw - 80rem) / -2);
-  --min-margin-left: calc(-100rem * 0.17);
+  --min-margin-left: calc(-100rem * 1.17);
```

Comment:
Try changing this back to how it was `--min-margin-left: calc(-100rem * 0.17);` to align the filters properly.

Reply:
Maybe it doesn't need to change to 1.17 again, but something in the middle. At the moment it's too close to the left border, almost as if there is no margin at all.